app/javascript/controllers/index.js

```diff
@@ -18,3 +18,6 @@ application.register("hello", HelloController)
 
 import MapController from "./map_controller"
 application.register("map", MapController)
+
+import NavbarController from "./navbar_controller"
+application.register("navbar", NavbarController)
```

Comment:
adds new navbar controller - on scroll down navbar disappears, on scroll up it re-appears.